package/src/types/general.ts

```diff
@@ -0,0 +1,16 @@
+type BaseRank = "Iron" | "Bronze" | "Silver" | "Gold" | "Platinum" | "Diamond" | "Ascendant" | "Immortal"
+type SubRank = "1" | "2" | "3"
+export type Rank = `${BaseRank} ${SubRank}` | "Radiant" | "Unrated"
+
+type Episodes = "1" | "2" | "3" | "4" | "5" | "6" | "7" | "8" | "9" | "10"
+type Acts = "1" | "2" | "3"
```

Comment:
```suggestion
type Episodes = number
type Acts = 1 | 2 | 3
```

Reply:
I think at that point, just make Season a string type, but I'll change it